.github/workflows/test.yml

```diff
@@ -49,11 +53,31 @@ jobs:
           export PATH="$HOME/.cargo/bin:$PATH"
           cargo install just
 
-      - name: Run all the tests
+      - name: Run all the tests (default)
+        if: ${{ !matrix.target.libwasmer_archive_name }}
         shell: bash
         run: |
           just test
 
+      - name: Run all the tests (with custom Wasmer runtime)
+        if: matrix.target.libwasmer_archive_name
+        shell: bash
+        run: |
+          libwasmer_url=$(curl -s https://api.github.com/repos/wasmerio/wasmer/releases/latest | jq --raw-output '.assets[] | select(.name == "${{ matrix.target.libwasmer_archive_name }}") | .browser_download_url')
```

Comment:
```suggestion
          libwasmer_url=$(curl -s https://api.github.com/repos/wasmerio/wasmer/releases/latest | jq --raw-output '.assets[] | select(.name == "wasmer-${{ matrix.target.id }}") | .browser_download_url')
```

Reply:
I prefer to not “connect” the matrix target ID with the archive name in this case.